petcare-sample/b2c/web-app/petdesk/web/react/src/pages/home.tsx

```diff
@@ -182,13 +182,13 @@ export const HomePage: FunctionComponent = (): ReactElement => {
         signOut();
     };
 
-    // If `clientID` is not defined in `config.json`, show a UI warning.
+    // If `clientID` is not defined in `config.js`, show a UI warning.
     if (!getConfig().clientID) {
 
         return (
             <div className="content">
                 <h2>You need to update the Client ID to proceed.</h2>
-                <p>Please open &quot;src/config.json&quot; file using an editor, and update
+                <p>Please open &quot;public/config.js&quot; file using an editor, and update
```

Comment:
public/config is used for only choreo
locally we are using config.js

Reply:
Config file is located inside the public folder. That is the meaning we want to convey to the user here.

Reply:
noted